pcdsdevices/dream_motion.py

```diff
@@ -44,3 +44,25 @@ class DREAM_SL3K4(BaseInterface, GroupDevice):
     bottom = Cpt(SmarAct, ':m12', kind='normal')
     north = Cpt(SmarAct, ':m9', kind='normal')
     south = Cpt(SmarAct, ':m8', kind='normal')
+
+
+class DREAM_Sample_Paddle(BaseInterface, GroupDevice):
+    """
+    DREAM Motion Class
+    This class controls sample paddle X,Y, Z, and Ret motors fixed to the DREAM Motion system for the
+    DREAM endstation in TMO.
+    Parameters
+    ----------
+    prefix : str
+        TMO:DREAM:MCS2:01
```

Comment:
Does this prefix example make sense?

Code looks good otherwise, just missing pre-release docs